packages/vscode-extension/src/debug/taskTerminal/localTunnelTaskTerminal.ts

```diff
@@ -24,12 +24,17 @@ import { doctorConstant } from "../depsChecker/doctorConstant";
 import { Step } from "../commonUtils";
 
 const ngrokTimeout = 1 * 60 * 1000;
-const defaultNgrokTunnelName = "bot";
-const ngrokEndpointRegex = (tunnelName: string) =>
-  new RegExp(`obj=tunnels name=${tunnelName} addr=(?<src>.*) url=(?<endpoint>.*)`);
+const ngrokTimeInterval = 10 * 1000;
+const ngrokEndpointRegex =
+  /obj=tunnels name=(?<tunnelName>.*) addr=(?<src>.*) url=(?<endpoint>https:\/\/([\S])*)/;
+const ngrokWebServiceRegex =
+  /msg="starting web service" obj=web addr=127.0.0.1:(?<webServicePort>\d+)/;
```

Comment:
Will it be `localhost:4040`, or it's always `127.0.0.1`?

Reply:
By default it is 127.0.0.1, but user can customize it in the config file. I will update the ngrokWebServiceRegex.
https://ngrok.com/docs/ngrok-agent/api
![image](https://user-images.githubusercontent.com/49138419/192469246-a470ca76-53f4-4ef6-81e4-a1335a3ff80c.png)